Source/SwiftLintFramework/Rules/TrailingWhitespaceRule.swift

```diff
@@ -46,6 +55,14 @@ public struct TrailingWhitespaceRule: CorrectableRule, ConfigurationProviderRule
         var corrections = [Correction]()
         let fileRegions = file.regions()
         for line in file.lines {
+            let commentKinds = SyntaxKind.commentKinds()
+            if line.content.hasTrailingWhitespace() && configuration.ignoresComments,
```

Comment:
Same with my comment on `validateFile(_:)`.
This loop does not need to evaluate following operations if `line.content.hasTrailingWhitespace()` returns `false`.
```swift
if !line.content.hasTrailingWhitespace() {
    correctedLines.append(line.content)
    continue
}
```